utils/_context/_scenarios/endtoend.py

```diff
@@ -38,6 +38,7 @@ def __init__(
         use_proxy=True,
         rc_api_enabled=False,
         meta_structs_disabled=False,
+        span_events=True,
```

Comment:
with the default value being `True`, it means that `SYSTEM_TESTS_AGENT_SPAN_EVENTS` will by default set to `true` in the proxy. So , and the default behavior never tested, right ?

Reply:
Correct, I did this so that the test doesn't depend on the agent version under test.

Reply:
My point is that currently : 

1. behaviour  when `span_events` is not set => never tested, do you confirm it's really what you want? By experience, it's the kind of stuff we really want to check ... The only reason not to test this is it cannot happen.
2. behaviour when  `span_events=false` => tested with `AGENT_NOT_SUPPORTING_SPAN_EVENTS` scenario, all good.
3. behaviour when  `span_events=true` => if you answer yes to 1. then it means it's the setup by default, so you can move all tests from `AGENT_SUPPORTING_SPAN_EVENTS` to the `default` scenario.

Reply:
> behaviour when span_events is not set

This will be tested by the parametrics tests added. They provided even deeper coverage than the e2e tests.